msi/GitProduct.wxs

```diff
@@ -46,6 +48,12 @@
           <Directory Id='BinFolder' Name='bin' />
         </Directory>
       </Directory>
+
+      <Directory Id='PersonalFolder' />
```

Comment:
Rarely is the default working folder what you want (IIRC): C:\Windows\system32 . Git is not system software, so I didn't think it should start the user running in the system folder.

It's called `PersonalFolder` because that's what the Windows Installer calls it: [PersonalFolder][pf] (I generally find this [MSDN doc][doc] more useful). Personally, I would have named it something else, probably: `DocumentsFolder` but they didn't ask me. :smile_cat: 

[pf]: https://msdn.microsoft.com/en-us/library/windows/desktop/aa370806%28v=vs.85%29.aspx
[doc]: https://msdn.microsoft.com/en-us/library/windows/desktop/aa370905%28v=vs.85%29.aspx#system_folder_properties